client/engine/messageservice/vectorclock_testmessageservice.go

```diff
@@ -76,23 +75,19 @@ func (t VectorClockTestMessageService) dispatchMessage(message protocols.Message
 // summarizeMessageSend returns a string which tersely summarizes the supplied message.
 // It may be used to make logs more readable.
 func summarizeMessageSend(msg protocols.Message) string {
-	summary := ""
+	str, _ := msg.Serialize()
+	size := len([]byte(str))
+	summary := fmt.Sprint(size) + "B:"
 	for _, entry := range msg.SignedProposals() {
```

Comment:
Should we update this to handle `Removal`s as well?

Reply:
Good call [25ae9b7](https://github.com/statechannels/go-nitro/pull/675/commits/25ae9b79cebf753ad8aa916aaa62352b43db6662)